ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/util/MapperUtil.java

```diff
@@ -1,19 +1,50 @@
 package uk.gov.hmcts.cmc.ccd.util;
 
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
 import java.util.Objects;
+import java.util.function.Function;
 import java.util.stream.Stream;
 
 public class MapperUtil {
+    private static final String OTHERS = " + others";
 
-    private MapperUtil(){
-        // Utility class, no instances
-    }
+    public static final Function<Claim, String> toCaseName = claim ->
+        fetchClaimantName(claim) + " Vs " + fetchDefendantName(claim);
 
-    public static boolean isAllNull(Object... objects) {
-        return Stream.of(objects).allMatch(Objects::nonNull);
+    private MapperUtil() {
+        // Utility class, no instances
     }
 
     public static boolean isAnyNotNull(Object... objects) {
         return Stream.of(objects).anyMatch(Objects::nonNull);
     }
+
+    private static String fetchDefendantName(Claim claim) {
+        StringBuilder defendantNameBuilder = new StringBuilder();
+
+        if (claim.getResponse().isPresent()) {
+            defendantNameBuilder
+                .append(claim.getResponse().map(response -> response.getDefendant().getName()).orElse(""));
+        } else {
+            defendantNameBuilder.append(claim.getClaimData().getDefendants().get(0).getName());
+        }
```

Comment:
Assuming the `orElse` above is just there to satisfy the `Optional` (our business rules should guarantee a response's defendant's name is present) this whole block can be rewritten:
```
claim.getResponse()
    .map(Response::getDefendant)
    .map(Party::getName)
    .orElseGet(() -> claim.getClaimData().getDefendants().get(0).getName())
```

Reply:
Will try this, seems correct.